daemon/call.c

```diff
@@ -4022,6 +4047,7 @@ int call_delete_branch(const str *callid, const str *branch,
 				"(via-branch '" STR_FORMAT_M "') in %d seconds",
 				STR_FMT_M(&ml->tag), STR_FMT0_M(branch), delete_delay);
 		ml->deleted = rtpe_now.tv_sec + delete_delay;
+		ml->mark_deleted = 1;
```

Comment:
I'm not 100% sure, but could `ml->deleted != 0` be used as a test for this instead of adding a new flag?

Reply:
As far as i remember i first tried it this way but there was an issue with it but i currently don't know what it was.
I need to make some tests and see, if i can  reproduce the issues i had.